CMakeLists.txt

```diff
@@ -25,37 +25,39 @@ else()
         QHotkey/qhotkey_p.h)
 endif()
 
-set(LIBS
-    Qt${QT_MAJOR}::Core
-    Qt${QT_MAJOR}::Widgets)
-
-set(SRC_FILES
+set(BASE_SRC_FILES
```

Comment:
May I ask you to do the same for source files as you did for libraries?
E.g. create the executable directly from `QHotkey/qhotkey.cpp` and use `target_sources` later for platform-depended libraries?

Reply:
Sure. I'll push it.

I don't know if I got it correctly. I've removed the variable and added the .cpp source file directly to `add_library()`.

Reply:
Great! But could you remove `PLATFORM_SRC_FILES` completely? I meant to replace `set` with `target_sources` as you did with `target_link_libraries`?

Reply:
Now I got it. Pushed.